### PR TITLE
feat: add prefix command

### PR DESCRIPTION
**Pull request by matfax**
This feature allows you to prefix paths.
Thereby, you can now move nested elements from one yaml to another, combining the read, prefix and merge commands.

### CODE REVIEW

**Comment by matfax** [comment]
I'll also update the docs.

**Comment by matfax** [comment]
There have been changes to mkdocs, please review them before merging.

**Comment by mikefarah** [comment]
wow this is cool - cheers :)

File: mkdocs/prefix.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+will output:
+```yaml
+something: else
+---
+c:
+  a: simple
+  b: cat
+```
+
+### Multiple Documents - update a single document
```
Comment (mikefarah) [inline mkdocs/prefix.md]: update all documents

**Comment by mikefarah** [comment]
LGTM - happy to merge and release once the prefix.md doc is fixed

**Comment by matfax** [comment]
@mikefarah I've changed it accordingly, I'm glad you appreciate it

**Pull request merged**